packages/components/tests/dummy/app/styles/showcase-pages/tabs.scss

```diff
@@ -0,0 +1,14 @@
+// TAB
+
+body.components-tabs {
+  .shw-component-tabs-layout-highlight {
+    .hds-tabs__tab {
+      outline: 1px dashed #78909c;
```

Comment:
Currently the outline is getting cut off. You could add this style to fix:
`outline-offset: -1px;`

Reply:
Good point! Fixed it :)